ofnetBgp.go

```diff
@@ -256,6 +288,8 @@ func (self *OfnetBgp) DeleteProtoNeighbor() error {
 //AddProtoNeighbor adds bgp neighbor
 func (self *OfnetBgp) AddProtoNeighbor(neighborInfo *OfnetProtoNeighborInfo) error {
 
+	<-self.start
```

Comment:
To avoid timing issue where the server start is not yet completed  (go routine). neighbor add would come in before that. Found during UT.